cmd/agent/container/credentials_server.go

```diff
@@ -99,31 +111,110 @@ func (cmd *CredentialsServerCmd) Run(ctx context.Context, _ []string) error {
 		return nil
 	}
 
-	binaryPath, err := os.Executable()
+	runnerAddr := checkRunnerCredentialServer(runnerPort)
 
+	// configure docker credential helper
+	if cmd.ConfigureDockerHelper && dockerCredentialsAllowed(runnerAddr) {
+		err = dockercredentials.ConfigureCredentialsContainer(cmd.User, port, log)
+		if err != nil {
+			return err
+		}
+	}
+
+	// configure git user
+	err = configureGitUserLocally(ctx, cmd.User, tunnelClient)
 	if err != nil {
-		return err
+		log.Debugf("Error configuring git user: %v", err)
```

Comment:
just making sure - we don't want to return here if it fails?

Reply:
forgot to change that back, will error now. Thanks!